coralproject/talk
# [CORL-1632] Allow basic auth config for scraper jobs

## What does this PR do?

Adds basic auth configuration for scraper jobs. If provided, scrape jobs will use a basic auth header with a provided username and password.

## What changes to the GraphQL/Database Schema does this PR introduce?

Adds `authorization`, `username`, and `password` to `StoryScrapingConfiguration` as well as its input type.

## How do I test this PR?

- Go to admin > config > advanced
- Enable authentication
- Set a username and password
- Do a scrape job
- Debug the scrap job and make sure it's setting the appropriate basic authorization header
